spec/spec_helper.rb

```diff
@@ -19,11 +19,26 @@
 
 # Code coverage analysis, output to coverage/ dir
 require 'simplecov'
+require 'webmock/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.